static/src/stylesheets/layout/new-header/_feedback-form.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Temporary form for feedback on the desktop navigation
+ */
+
+.feedback-form {
```

Comment:
Don't worry, this isn't merged yet. I will add this today - it shouldn't take long. Thanks!